go.sum

```diff
@@ -729,6 +729,16 @@ github.com/hpcloud/tail v1.0.0/go.mod h1:ab1qPbhIpdTxEkNHXyeSf5vhxWSCs/tWer42PpO
 github.com/hyperledger/firefly-common v0.1.13/go.mod h1:2NqPi5Ud9H6rSlZXkLbotxW7z4EAD89p3/8oNOpm9Gs=
 github.com/hyperledger/firefly-common v0.1.16 h1:21xidDEKrJhtGdBSRqHN4PfDi7aYxF0HOFuAa04V1AE=
 github.com/hyperledger/firefly-common v0.1.16/go.mod h1:MNbaI2spBsdZYOub6Duj9xueE7Qyu9itOmJ4vE8tjYw=
+github.com/hyperledger/firefly-common v0.1.19-0.20220811192657-e63ab6827844 h1:DDWL4Tv2vZbcKgc/1dSjgei0QnWIQnu3ikRqg7tBnkQ=
+github.com/hyperledger/firefly-common v0.1.19-0.20220811192657-e63ab6827844/go.mod h1:MNbaI2spBsdZYOub6Duj9xueE7Qyu9itOmJ4vE8tjYw=
+github.com/hyperledger/firefly-common v0.1.19-0.20220811194331-db95d0f1bc31 h1:ijIhHR9uDwGBXOBydaQxAVyyReNmUWcTHW5R6oJYc5U=
+github.com/hyperledger/firefly-common v0.1.19-0.20220811194331-db95d0f1bc31/go.mod h1:MNbaI2spBsdZYOub6Duj9xueE7Qyu9itOmJ4vE8tjYw=
+github.com/hyperledger/firefly-common v0.1.19-0.20220811195438-ef727fa10629 h1:id4k0rBHXzcez+1qy+E8xP4J3ZudNtuSF7dJaBhPqMI=
+github.com/hyperledger/firefly-common v0.1.19-0.20220811195438-ef727fa10629/go.mod h1:MNbaI2spBsdZYOub6Duj9xueE7Qyu9itOmJ4vE8tjYw=
+github.com/hyperledger/firefly-common v0.1.19-0.20220811200324-b6c4c44d2d0d h1:9s1MI4RIfKgftrGQ6sI2gvQBEQElq2CQOs3uaiB44s8=
+github.com/hyperledger/firefly-common v0.1.19-0.20220811200324-b6c4c44d2d0d/go.mod h1:MNbaI2spBsdZYOub6Duj9xueE7Qyu9itOmJ4vE8tjYw=
+github.com/hyperledger/firefly-common v0.1.20 h1:0dShkjlIShyBxkXRmu3vLmpEK6xrqmfc8GhF6k0Vgbg=
+github.com/hyperledger/firefly-common v0.1.20/go.mod h1:gMlv4Iy5JjnzXmSEdb+tWVDIc/2GhL9MRcgNX+VmI4M=
```

Comment:
I don't understand go.sum but why are there so many firefly-common entries??

Reply:
I was testing along the way - it seems to just add in each one you use, and never remove it

Reply:
e.g. I was doing `go get github.com/hyperledger/firefly-common@b6c4c44d2d0dc4b1c511aab629facf67113e3708` from branches in common, to test my changes

Reply:
`go mod tidy`?

Reply:
Decision: Will do this in next PR, to keep the pipeline moving as this is way into the E2Es